portfolio/index.html

```diff
@@ -0,0 +1,214 @@
+                            <h3>T-Shirt Form</h3>
+                            <p>This is a basic T-shirt order form built using HTML and CSS.</p>
+                            <a href="https://form-control-html.netlify.app/" class="project-link" target="_blank">View
+                                on
+                                Netlify</a>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="project-card">
+                    <div class="project-image">
+                        <img src="https://module765432.netlify.app/img/first-background.jpg" alt="Project 2">
+                        <div class="project-info">
+                            <h3>Karma Device Store</h3>
+                            <p>A stylish and responsive online store for Karma Devices, built with HTML and CSS.</p>
+                            <a href="https://module765432.netlify.app/" class="project-link" target="_blank">View on
+                                Netlify</a>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="project-card">
+                    <div class="project-image">
+                        <img src="https://static.wixstatic.com/media/84770f_30b5b37a554c487d85fffde598298a77~mv2.jpg/v1/crop/x_429,y_0,w_1514,h_1900/fill/w_322,h_404,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/Website_Mockup.jpg"
+                            alt="Project 3">
+                        <div class="project-info">
+                            <h3>A Multi-page clone Project</h3>
+                            <p>A multipage clone project designed to create a professional CV. Built using HTML and CSS.
+                            </p>
+                            <a href="https://multi-page-html.netlify.app/" class="project-link" target="_blank">View on
+                                Netlify</a>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="project-card">
+                    <div class="project-image">
+                        <img src="https://t3.ftcdn.net/jpg/03/74/73/78/360_F_374737839_VsxlmnN7soqOOOFFzy5kfE9H6PFMCkPX.jpg"
+                            alt="Project 4">
+                        <div class="project-info">
+                            <h3>Online Food Store</h3>
+                            <p>An online food store built with HTML ,CSS and CYF Blocks</p>
+                            <a href="https://effervescent-sawine-280009.netlify.app/" class="project-link"
+                                target="_blank">View on
+                                Netlify</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="contact-form-section" id="contact">
+            <h1>Contact Me</h1>
+            <form class="contact-form" action="submit_form.php" method="POST">
```

Comment:
Shall we just insert a google form for now?
Or you wanted to develop your own?
Not sure what's better. Google form just works and shows that you're pragmatic.
Your own form gives an experience and shows that you can write a backend.
Maybe I overthink.
Just wanted to consider the option.

Reply:
I think it's a great idea to use Google Forms for now. I haven’t used it before, so this would be a good opportunity to learn how it works. That’s why I didn’t consider it earlier, but I agree it’s a practical solution and not a bad idea at all.